pkg/service/service.go

```diff
@@ -187,13 +190,76 @@ func (s *Service) GetCurrentTs() time.Time {
 	return time.Now()
 }
 
+func (s *Service) populateBackendMapV1FromV2(ipv4, ipv6 bool) error {
+	removeMap := func(m string) {
```

Comment:
Have you considered replacing `removeMap` with `(*bpf.Map).Unpin()`?

Reply:
Good point, done :)